mlflow/server/js/src/components/ParallelCoordinatesPlotControls.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { TreeSelect } from 'antd';
+
+export class ParallelCoordinatesPlotControls extends React.Component {
+  static propTypes = {
+    paramKeys: PropTypes.arrayOf(String).isRequired,
+    metricKeys: PropTypes.arrayOf(String).isRequired,
+    selectedParamKeys: PropTypes.arrayOf(String).isRequired,
+    selectedMetricKeys: PropTypes.arrayOf(String).isRequired,
+    handleParamsSelectChange: PropTypes.func.isRequired,
+    handleMetricsSelectChange: PropTypes.func.isRequired,
+  };
+
+  static handleFilterChange = (text, option) =>
+    option.props.title.toUpperCase().includes(text.toUpperCase());
```

Comment:
What does this thing do?

Reply:
It's a function to define the filter behavior when you type in characters. Here we are saying that nodes are filtered cases insensitively. 
https://ant.design/components/tree-select/#Tree-props